deployments/kubehound/docker-compose.release.ingestor.yaml

```diff
@@ -0,0 +1,12 @@
+name: kubehound-release
+services:
+  grpc:
+    image: ghcr.io/datadog/kubehound-ingestor:latest
```

Comment:
same comment about pinned version, I thought we wanted to have latest for dev but pinned version for releases

Reply:
We pinned for the embeded one, I think for the ingestor it is fine to use latest. Same I dont want to chase the new upgrade. As it is also for example.